src/firefly/js/templates/lightcurve/LcUtil.jsx

```diff
@@ -16,6 +16,23 @@ export function getTypeData(key, val='', tip = '', labelV='', labelW) {
     };
 }
 
+/**
+ * Returns only numerical column names form raw lc table
+ * @param {Object} rawTbl
+ * @returns {TableColumn[]} - array of table columns objects
+ */
+export function getOnlyNumericalCol(rawTbl) {
+
+    var colType = ['double', 'd', 'long', 'l', 'int', 'i', 'float', 'f'];
+    return get(rawTbl, ['tableData', 'columns']).reduce((prev, col) => {
+        if ((colType.includes(col.type)) &&
+            (!has(col, 'visibility') || get(col, 'visibility') !== 'hidden')) {
+            prev.push(col.name);
+        }
+        return prev;
```

Comment:
These sort of functions should be in TableUtil.js.  It would also be easier to read if written as:
```
        return get(rawTbl, ['tableData', 'columns'])
                .filter((col) => colType.includes(col.type) && get(col, 'visibility') !== 'hidden')
                .map((col) => col.name);


```